src/subapps/search/views/GlobalSearchView.tsx

```diff
@@ -177,8 +435,9 @@ const constructQuery = (searchText: string) => {
       prefix_length: 0,
       fields: ['*'],
     })
-    .size(50)
-    .from(0);
+    .rawOption('track_total_hits', true) // accurate total, could cache for performance
+    .size(pageSizeToNotExceedLimit)
```

Comment:
this will add the from and size to the query payload and not to the URI endpoint, right?

Elasticsearch allows [both](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-from-size.html), to define it on the payload and on the URI (?from={offset}&size={size}).

Currently we enforce `from` and `size` only using query parameters. So If you are going to provide it using the payload, we will have an extra work to do on the backend side to validate those fields (which is fine, but we'll need to plan for it).